android/src/com/mapswithme/maps/routing/RoutingPlanController.java

```diff
@@ -108,18 +127,29 @@ public void onClick(View v)
       }
     });
 
+    setupRouterButton(R.id.taxi, R.drawable.ic_taxi, new View.OnClickListener()
+    {
+      @Override
+      public void onClick(View v)
+      {
+        AlohaHelper.logClick(AlohaHelper.ROUTING_TAXI_SET);
```

Comment:
Yeah, I understand your proposal. But I think if we decide to move this logic to the core then it should be done for all route types (pedestrian, bycicle and auto) , **not only ** for taxi route. I wanted to  say that was trying to be consistent with existing way to work with sending statistics data. If we want to change this way we should change it, but it should be done globally I guess, not only for this concrete case. Also I should figure out why aloha calls are made in Java code first. I would propose to discuss this out of this PR, because It's really not related to this feature.